### PR TITLE
Update Anomaly OpenVINO export for ModelAPI

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
### Summary

Updates keys in `rt_info` for modelAPI. Since min and max values are used for normalization, both are fused into one. This also reverses the input channles so that RGB image is passed to the model during inference by modelAPI

### CODE REVIEW

File: src/otx/algorithms/anomaly/tasks/inference.py [hunk 1]
```diff
@@ -333,21 +333,29 @@ def _add_metadata_to_ir(self, xml_file: str) -> None:
...
+                    )
                 elif transform == "Resize":
                     model.set_rt_info(transform_dict["height"], ["model_info", "orig_height"])
                     model.set_rt_info(transform_dict["width"], ["model_info", "orig_width"])
                 else:
                     warn(f"Transform {transform} is not supported currently")
+        # Since we only need the diff of max and min, we fuse the min and max into one op
+        if "min" in metadata and "max" in metadata:
+            model.set_rt_info(metadata["max"] - metadata["min"], ["model_info", "normalization_scale"])
+        model.set_rt_info(True, ["model_info", "reverse_input_channels"])
```
Comment (sovrasov) [inline src/otx/algorithms/anomaly/tasks/inference.py]: I propose using `embed_ir_model_data` here to unify implementation with other tasks.

**Review by sovrasov — CHANGES_REQUESTED** [review]

File: src/otx/algorithms/anomaly/tasks/inference.py (hunk 1 above)
Comment (ashwinvaidya17) [reply src/otx/algorithms/anomaly/tasks/inference.py]: Thanks. Didn't know it existed. That's pretty neat!

Comment (ashwinvaidya17) [reply src/otx/algorithms/anomaly/tasks/inference.py]: Done :)

**Review by sovrasov — COMMENTED** [review]
Could you also enable flag `check_ir_meta` in `otx_export_testing` for anomaly models? 
`otx_export_testing` checks that `labels` field exists, and you may add labels=[] to metadata, if we don't need them in case of anomaly.

**Comment by goodsong81** [comment]
@ashwinvaidya17 I've refined the title according to the convention.

**Review by sovrasov — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**